plugins/module_utils/database.py

```diff
@@ -0,0 +1,189 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Do we need to add a large note here (and in community.general) to keep these files in sync?

Reply:
Does this file also exist in community.proxysql?

Reply:
no, module_utils/mysql also exists in mysql.  this file had shared content for postgresql and mysql in community.general, there's no need to keep this going forward, I've just ported it across to minimise change on the initial commit.

Reply:
when we get past the deadline of this week for breaking these out of community.general I can start focusing on the actual content 👍